pkg/storageos/deploy_test.go

```diff
@@ -486,6 +486,7 @@ func TestCreateCSIHelper(t *testing.T) {
 	}
 }
 
+// NOTE: Unsupported legacy test.
```

Comment:
Will remove all the legacy tests when making CSI the default in cluster config separately.